include/autocluster.hrl

```diff
@@ -28,6 +28,7 @@
          {config, cluster_name,          "CLUSTER_NAME",           "undefined",  string,  false}, %% Consul && etcd
 
          {config, consul_acl_token,      "CONSUL_ACL_TOKEN",       "undefined",  string,  false}, %% Consul
+         {config, consul_allow_warn,     "CONSUL_ALLOW_WARN",      false,        atom,    false},
```

Comment:
I'd like to find a more descriptive name. `CONSUL_TOLERATE_WARN` or `CONSUL_ALLOW_WARN` seem too short to me. It's not clear what are we allowing here.

I'd go with something like `CONSUL_INCLUDE_NODES_WITH_WARNINGS`. While longer, it is more or less to the point. WDYT?